spec/unit/util/windows/sid_spec.rb

```diff
@@ -15,10 +15,15 @@
   context "#octet_string_to_sid_object" do
     it "should properly convert an array of bytes for the local Administrator SID" do
       host = '.'
-      # Bringing the ADSI module in isn't ideal, but we're leaving re-arranging the modules
-      # for a future breaking release.
-      username = Puppet::Util::Windows::ADSI::User.current_user_name
-      admin = WIN32OLE.connect("WinNT://#{host}/#{username},user")
+      # Get the first user in the Administrators group
+      admins = WIN32OLE.connect("WinNT://#{host}/Administrators,group")
+      admin = nil
+      admins.members.each do |member|
+        admin = member
+        break
+      end
+      username = admin.name
+
```

Comment:
It's sad that WIN32OLE objects behave like enumerable objects but don't implement Enumerable, so you can't just do `admins.members.first`. That said this test is just making sure `octet_string_to_sid_object` works, so how about if we just specify a byte array for an account we know exists, e.g.

```
C:\work\puppet [stable +9 ~0 -0 !]> bundle exec irb
irb(main):001:0> require 'puppet'
=> true
irb(main):002:0> sid = Win32::Security::SID.new('SYSTEM').sid.unpack('C*')
=> [1, 1, 0, 0, 0, 0, 0, 5, 18, 0, 0, 0]
irb(main):003:0> Puppet::Util::Windows::SID.octet_string_to_sid_object(sid)
=> #<Win32::Security::SID:0x39750f0 @sid="\x01\x01\x00\x00\x00\x00\x00\x05\x12\x00\x00\x00", @account="SYSTEM", @host="WIN-QP47VOHA2P4", @domain="NT AUTHORITY", @account_type="well known group">
```

So we could simplify the test to:

```
sid =  [1, 1, 0, 0, 0, 0, 0, 5, 18, 0, 0, 0]
converted = subject.octet_string_to_sid_object(sid)
expect(converted).to eq(Win32::Security::SID.new('SYSTEM', host))
```